packages/teleport/src/Apps/AddApp/Manually/Manually.tsx

```diff
@@ -60,16 +60,30 @@ export default function Manually({ version, onClose }: Props) {
             {' - Generate a join token'}
             <TextSelectCopy mt="2" text="tctl tokens add --type=app" />
           </Box>
-          <Box>
+          <Box mb="4">
             <Text bold as="span">
               Step 4
             </Text>
-            {` - Install Teleport on target server, and start it with the following parameters`}
+            {` - Start the Teleport agent with the following parameters`}
             <TextSelectCopy
               mt="2"
-              text={`teleport start --roles=app --token=<generated-app-join-token> --auth-server=${cfg.proxyCluster} `}
+              text={`teleport start --roles=app --app-name=[example-app] --app-uri=http://localhost:8080 --token=[generated-node-join-token] --auth-server=${host} `}
             />
           </Box>
+          <Box>
+            <Text bold as="span"></Text>
+            {`* Note, for a self-hosted Teleport version, you may need to update DNS and obtain a TLS certificate for this application.
+            Learn more about application access `}
+            <Link
+              href={
+                'https://gravitational.com/teleport/docs/application-access/'
+              }
+              target="_blank"
+              mr="2"
+            >
+              here
```

Comment:
Instead of the word `here`, i think its better to just link `application access` without the `here`. `application access` will be underlined, and colored, so that implies its a link